src/app/components/Common/NavigationBar/Search/SearchBar.tsx

```diff
@@ -132,7 +132,8 @@ class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
 
     const catalogItems = catalogData && catalogData[type] ? catalogData[type].data : [];
 
-    return [...libraryItems, ...catalogItems];
+    // Filter out invalid items returned from the API.
+    return [...libraryItems, ...catalogItems].filter(item => item.attributes !== undefined);
```

Comment:
I've seen some occurrences where the `attributes` property is present, but `null`, the filter should read `(item => !!item.attributes)`